keycloak/keycloak-operator
# KEYCLOAK-15014 Add scopes to realm and client

## JIRA ID
https://issues.redhat.com/browse/KEYCLOAK-15014

## Additional Information
Supporting scopes is nice to be able to add additional claims to JWT tokens.

**This PR builds on other PRs, only the last commit is relevant for this change**

## Verification Steps

The REST API is tested using e2e import/export.

Manual testing is possible:

1. Create a CR with a custom scope adding a static attribute and all other required mappers
2. Log in and verify the claims

## Checklist:
